docs/contrib/GettingInvolved.md

```diff
@@ -0,0 +1,104 @@
+# Getting Involved
```

Comment:
We should have a preamble on both docs explaining the scope of each repos and detail the ways to add contributions for both. Toosl --> OR; PDK and designs --> ORFS, etc.